pywt/_extensions/_pywt.pyx

```diff
@@ -85,7 +85,7 @@ class _Modes(object):
     --------
     >>> import pywt
     >>> pywt.Modes.modes
-        ['zero', 'constant', 'symmetric', 'reflect', 'periodic', 'smooth', 'periodization', 'antisymmetric', 'antireflect']
+         ['zero', 'constant', 'symmetric', 'periodic', 'smooth', 'periodization', 'reflect', 'antisymmetric', 'antireflect']
```

Comment:
minor: added indentation looks off - shouldn't this be de-indented to start at the same place as the first `>` on the line above it?

Reply:
Done.
Doctester does not fail either way because it normalizes whitespace.